pcdsdevices/attenuator.py

```diff
@@ -28,6 +29,34 @@
 MAX_FILTERS = 12
 
 
+class GattApertureY(TwinCATStatePMPS):
+    """
+    AT1K0 is a gas attenuator containing 4 discrete aperture arrays,
```

Comment:
Much more helpful to the uneducated (like me 😄 ). Is this restricted to AT1K0?  Or are there similar devices that might use this class?

I only ask because I've seen device classes for explicit variants (like [AT2L0](https://github.com/pcdshub/pcdsdevices/blob/f7d03f4aaa3f84512c2ef6e38944a0ae0e0c0de4/pcdsdevices/attenuator.py#L1047)).  If this is a general use class, I'd just suggest writing the docstring to suggest that.  (mentioning AT1K0 as an example implementation rather than **_the_** device, maybe)

Reply:
ah okay, to my understanding, this is the only place on the beamline that this particular style of aperture of is used.  I believe a different class would be used or written for any other device.

Reply:
Gotcha, sounds good then.  Thanks for humoring my naïveté